File: go/get_destination_uris.go
package main

import (
    "fmt"
    "os"
)

func main() {
    trunkSid := os.Getenv("TRUNK_SID")
    if trunkSid == "" {
        fmt.Println("Error: TRUNK_SID environment variable is required")
        os.Exit(1)
    }

    fmt.Printf("Getting destination URIs for trunk %s...\n", trunkSid)
    result := get(fmt.Sprintf("/trunks/%s/destination-uris", trunkSid))
    fmt.Println("Destination URIs retrieved successfully!")
    _ = result
} 

File: php/get_destination_uris.php
#!/usr/bin/env php
<?php

require_once '_client.php';

// Get destination URIs for a trunk
$trunk_sid = getenv('TRUNK_SID');
if (!$trunk_sid) {
    echo "Error: TRUNK_SID environment variable is required\n";
    exit(1);
}

echo "Getting destination URIs for trunk $trunk_sid...\n";
$result = exo_get("/trunks/$trunk_sid/destination-uris");
echo "Destination URIs retrieved successfully!\n";
?> 